pkg/scaffold/scaffold.go

```diff
@@ -185,14 +166,19 @@ func (s *Scaffold) Execute(u *model.Universe, options input.Options, files ...in
 		}
 	}
 
-	if u.Boilerplate == "" {
-		u.Boilerplate = s.Boilerplate
-	}
-
 	if err := s.defaultOptions(&options); err != nil {
 		return err
 	}
 
+	// Set default values for the universe
+	if u.Project == nil {
+		u.Project = &s.Project
+	}
+	if u.Boilerplate == "" {
+		u.Boilerplate = s.Boilerplate
+	}
+	u.Files = make([]*model.File, 0, len(files))
```

Comment:
Since you add a comment to described wdyt about extract it a func? E,g

```go
setDefaultValuesUniverse()
...
func setDefaultValuesUniverse() {
	if u.Project == nil {
		u.Project = &s.Project
	}
	if u.Boilerplate == "" {
		u.Boilerplate = s.Boilerplate
	}
	u.Files = make([]*model.File, 0, len(files))
}
```